src/index.js

```diff
@@ -7,7 +7,7 @@
 import {Plugin} from "prosemirror-state"
 
 import {handleTripleClick, handleKeyDown, handlePaste, handleMouseDown} from "./input"
-import {key} from "./util"
+import {key, cellAround, inSameTable} from "./util"
```

Comment:
I think this change is not needed anymore, look at https://github.com/ProseMirror/prosemirror-tables/pull/40/files#diff-1fdf421c05c1140f6d71444ea2b27638L69

Reply:
Completely agree, thanks for pointing this out. These two imports have been removed from the PR.